node/pkg/processor/vaa.go

```diff
@@ -8,6 +8,7 @@ import (
 type VAA struct {
 	vaa.VAA
 	Unreliable bool
+	IsReobserv bool
```

Comment:
can we call this `Reobservation` , can leave IsReobservation method? I think it fits like `Unreliable` and `IsReliable`